### ISSUE TITLE
can extensions also add dependencies to the build.gradle

### ISSUE THREAD

**Issue by cpapplefamily** [wpilibsuite/RobotBuilder#149]
Similar to issue #130 could extensions have fields to add repositories and dependencies to the build.gradle?

**Comment by ThadHouse** [comment]
I would be ok with them having a way to add vendor deps, but I would be scared of modifying build.gradle directly. It's way too easy to screw up, and too hard to parse correctly to get right.

**Comment by cpapplefamily** [comment]
Do you have any leads I can follow on how to create a vendor dept? I have successfully imported my Libraries using the build.gradle  but not vendor dep. Omittingly I'm a power hacker not a power user here.
